### PR TITLE
Bump libsigsegv to 2.11

### PR DESCRIPTION
**Pull request by alarcher**
Bump to non-ABI-compatible, postponed.
> pkg search -r -o pkg.name 'depend:require:library/libsigsegv'
PKG.NAME
runtime/clisp
developer/macro/gnu-m4
text/gnu-diffutils
text/gawk

### CODE REVIEW

File: components/library/libsigsegv/Makefile [hunk 1]
```diff
@@ -19,33 +19,47 @@
...
 
 install:	$(INSTALL_32_and_64)
 
 test:		$(TEST_32_and_64)
 
-BUILD_PKG_DEPENDENCIES =	$(BUILD_TOOLS)
-
-include ../../make-rules/depend.mk
+# Auto-generated dependencies
+REQUIRED_PACKAGES += system/library
```
Comment (pyhalov) [inline components/library/libsigsegv/Makefile:67]: Please, remove resolve.deps

File: components/library/libsigsegv/test/results-64.master [hunk 2]
```diff
@@ -0,0 +1,12 @@
+PASS: sigsegv1
```
Comment (pyhalov) [inline components/library/libsigsegv/test/results-64.master]: If files don't differ, deliver one file

**Comment by pyhalov** [comment]
Also 
text/gawk

**Comment by pyhalov** [comment]
I'd say, bump and rebuild.

File: components/library/libsigsegv/Makefile (hunk 1 above)
Comment (alarcher) [reply components/library/libsigsegv/Makefile:67]: Done.

**Comment by alarcher** [comment]
I added packages to be rebuilt in #3234

**Pull request merged**